lib/modules/platform/codecommit/index.spec.ts

```diff
@@ -0,0 +1,1100 @@
+import {
+  CodeCommitClient,
+  CreatePullRequestApprovalRuleCommand,
+  CreatePullRequestCommand,
+  DeleteCommentContentCommand,
+  DescribePullRequestEventsCommand,
+  GetCommentsForPullRequestCommand,
+  GetFileCommand,
+  GetPullRequestCommand,
+  GetRepositoryCommand,
+  ListPullRequestsCommand,
+  ListRepositoriesCommand,
+  MergeBranchesBySquashCommand,
+  PostCommentForPullRequestCommand,
+  UpdatePullRequestDescriptionCommand,
+  UpdatePullRequestStatusCommand,
+  UpdatePullRequestTitleCommand,
+} from '@aws-sdk/client-codecommit';
+import { GetUserCommand, IAMClient } from '@aws-sdk/client-iam';
+import { mockClient } from 'aws-sdk-client-mock';
+import { TextEncoder } from 'web-encoding';
+import {
+  PLATFORM_BAD_CREDENTIALS,
+  REPOSITORY_EMPTY,
+  REPOSITORY_NOT_FOUND,
+} from '../../../constants/error-messages';
+import type { logger as _logger } from '../../../logger';
+import { PrState } from '../../../types';
+import * as git from '../../../util/git';
+import type { Platform } from '../types';
+import { config } from './index';
+
+describe('modules/platform/codecommit/index', () => {
+  let codeCommit: Platform;
+  let logger: jest.Mocked<typeof _logger>;
+  let codeCommitClient: any;
+  let iamClient: any;
```

Comment:
```suggestion
  let iamClient: mockClient(IAMClient);
```

do it once, so all new instances are mocked instances

Reply:
you mean  `const iamClient = mockClient(IAMClient);`